docs/tutorials/using_apis/connectivity/mesh_tutorial.md

```diff
@@ -0,0 +1,136 @@
+<h1 id="mesh-tutorial">Mesh tutorial</h1>
+
+## Introduction
+
+Mbed OS supports variety of 802.15.4 based Mesh protocols. There is support for [Wi-SUN](../reference/wisun-tech.html), [Thread](../reference/thread-tech.html) and [6LoWPAN-ND](../reference/6LoWPAN-ND-tech.html) protocols and they all have their own unique characteristics. For example, Thread is designed for Home automation use while Wi-SUN and 6LoWPAN-ND are targeted more for Utilities and Smart metering. In addition, Wi-SUN and Thread are having certification programs to ensure interoperability between devices from different vendors.
+
+
+## Mesh application design principles
```

Comment:
How does this section relate to what this tutorial is trying to accomplish?

Reply:
We didn't find any better place for "Mesh App Design guidelines". So far they are not documented anywhere and we feel that they could be part of Mesh tutorial.  Would you have any suggestions for the better location?